lib/resources/member-saved-items.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * Member Saved Items resources
+ *
+ * @module resources/items
+ * @copyright 2015–2024 RewardOps Inc.
+ */
+
+const config = require('../config');
+const api = require('../api');
+
+/**
+ * Higher order function for creating a `getAll` program items function.
+ *
+ * @param {number} programId The ID of the item's parent program
+ *
+ * @returns {module:resources/items~GetAllFunc} Returns `getAll` program items function.
+ *
+ * @see {@link module:resources/items~GetAllFunc} for examples.
+ *
+ * @protected
+ */
+const getAllProgramItems = programId => {
+  return function getAll(params, callback) {
+    const options = {
+      path: `/programs/${programId}/member_saved_items`,
+      config: config.getAll(),
+    };
+
+    /*
+     * If called with two arguments,
+     * set options.params to the first
+     * argument (`params`).
+     *
+     * Otherwise (i.e., called with just
+     * one argument), set the callback as
+     * the argument.
+     */
+    if (arguments.length === 2) {
+      options.params = params;
+    } else {
+      // eslint-disable-next-line no-param-reassign
+      callback = params;
+    }
+
+    api.get(options, callback);
+  };
+};
+
+/**
+ * Function for getting an array of item JSON objects.
+ *
+ * @typedef module:resources/items~GetAllFunc
+ *
+ * @property {object} params [Request]{@link https://github.com/request/request} params to pass to the API call
+ * @property {module:api~requestCallback} callback Callback that handles the response
+ *
+ * @example
+ * // Gets an array of item detail objects the program with id 12
+ * ro.program(12).items.getAll((error, responseBody, response) => {
+ *   // ...
+ * });
+ */
+
+/**
+ * Higher order function for creating a `get` program item function.
+ *
+ * @param {number} programId The ID of the item's parent program
+ *
+ * @returns {module:resources/items~GetFunc} Returns `get` program item function.
+ *
+ * @see {@link module:resources/items~GetFunc} for examples.
+ *
+ * @protected
+ */
+const getProgramItem = programId => {
+  return function get(itemId, params, callback) {
+    const options = {
+      path: `/programs/${programId}/member_saved_items/${itemId}`,
```

Comment:
@k163791 this Pangea API endpoint doesn't exist either. Thanks for asking about this @imlytvynenko

Reply:
@DanielStevenLewis have removed those endpoints.